whois-api/src/test/java/net/ripe/db/whois/api/rest/InternalUpdatePerformerTest.java

```diff
@@ -186,7 +188,7 @@ public void setSsoSessionToContext_successful_sso_translation() {
 
     @Test
     public void setSsoSessionToContext_exception_is_logged() {
-        when(ssoTokenTranslatorMock.translateSsoToken("test-token")).thenThrow(new CrowdClientException("exception"));
+        when(ssoTokenTranslatorMock.translateSsoToken("test-token")).thenThrow(new CrowdClientException(Response.Status.INTERNAL_SERVER_ERROR.getStatusCode(), "exception"));
```

Comment:
Use `HttpStatus.INTERNAL_SERVER_ERROR_500`

Reply:
Authclient is part of whois-client module and we do not have org.eclipse.jetty. In this module we use javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR in other classes as well